### ISSUE TITLE
Expose client list or add messageExclude and binaryExclude

### ISSUE THREAD

**Issue by benkuper** [me-no-dev/ESPAsyncWebServer#1347]
Hello,
this has been already asked (and apparently ignored) here : #466 and #483

When having a bi-directional communication for parameter synchronisation for instance, you often don't want to send the value update feedback to the connection that the modification comes from.
With this very useful and standard paradigm, methods like messageExclude and sendExclude with providing either one id  or an array of id to exclude when sending to all would be super useful.

In other words : 
Clients C1 , C2 and C3 connected to the WS server. C1 sends value update > WS receives and updates the value > parameter update sends feedback to C2 and C3 but not C1.

Alternatively, if this is for some reason something that shouldn't make it in the official implementation, giving access to the clients list would help doing it outside the class (public), or by creating a child class and add the methods (protected)

Thanks
